src/api/spec/models/user_spec.rb

```diff
@@ -870,4 +870,22 @@
       end
     end
   end
+
+  describe '#autocomplete' do
+    let!(:foobar) { create(:confirmed_user, login: 'foobar') }
+    let!(:fobaz) { create(:confirmed_user, login: 'fobaz') }
+
+    context 'autocomplete' do
+      subject { User.autocomplete('foo') }
+
+      it { expect(subject).to include('foobar') }
```

Comment:
In the `include` cases I would prefer to use `match_array` that checks exactly what values the array have, because you want to know which values are retrieved, not only that the value you are expecting is there.

Reply:
:+1: to  @mdeniz, then you can remove the `not_to include` case. :tada: